packages/design-system/src/components/Review/Review.tsx

```diff
@@ -12,7 +11,7 @@ export interface ReviewProps {
   children: React.ReactNode;
   className?: string;
   /**
-   * Optional label to give screenreaders longer, more descriptive text to
+   * Optional label to give screen readers longer, more descriptive text to
    * explain the context of an edit link.
    */
   editAriaLabel?: string;
```

Comment:
I didn't change this prop as it might break the API and I don't wanna open a can of worms for this small ticket, but do have questions about how this component might pass a11y testing.

I'm thinking, if there are several Review components in a row (as we have in [one example](https://design.cms.gov/storybook/?path=/story/patterns-review--multiple-reviews&globals)) and all Reviews have the default "Edit" link, **do we run into the a11y issue of having multiple links with the same label?**

Should we make an aria-label required? Or provide a better fallback? I was initially thinking a good fallback could be setting an `aria-label=${editText} ${heading}`, but `heading` is also optional. Anyways, this is a longwinded way to say we might need a follow-up ticket to address this issue (if it is one).

cc: @SparrowSpit

Reply:
In a real use-case scenario this is used at the end of the healthcare application, right? If so, I'm pretty sure what we have going on there is `aria-labelledy={heading}` where it points to the heading's ID but I'd have to triple check to be 100% on that. 

That accomplishes what you're getting at where we're "connecting" the two elements and providing contextual info for assistive tech.